test/propagated_params_test.go

```diff
@@ -44,7 +44,8 @@ var (
 	ignoreConditions        = cmpopts.IgnoreFields(duckv1.Status{}, "Conditions")
 	ignoreContainerStates   = cmpopts.IgnoreFields(corev1.ContainerState{}, "Terminated")
 	ignoreStepState         = cmpopts.IgnoreFields(v1beta1.StepState{}, "ImageID")
-	ignoreTaskRunSpec       = cmpopts.IgnoreFields(v1beta1.TaskRunSpec{}, "Resources")
+	ignoreTaskRunSpec       = cmpopts.IgnoreFields(v1beta1.TaskRunSpec{}, "Resources", "ServiceAccountName")
+	ignorePipelineRunSpec   = cmpopts.IgnoreFields(v1beta1.PipelineRunSpec{}, "ServiceAccountName")
```

Comment:
suggest renaming to "ignoreServiceAccountName" and adding a comment that we're ignoring it because the default can differ between platforms